build.sbt

```diff
@@ -16,8 +16,8 @@ libraryDependencies ++= Seq(
   "com.gu" %% "scanamo" % "1.0.0-M4"
 )
 
- topLevelDirectory in Universal := None
- packageName in Universal := normalizedName.value
+Universal / topLevelDirectory := None
+Universal / packageName := normalizedName.value
```

Comment:
The ` in ` syntax became deprecated with the introduction of sbt 1.0: https://www.scala-sbt.org/1.x/docs/Migrating-from-sbt-013x.html#Migrating+to+slash+syntax